src/index.ts

```diff
@@ -119,22 +119,36 @@ module.exports = class ServerlessPlugin {
   }
 
   private addServiceAndEnvTags() {
```

Comment:
suggestion: it would be helpful to have a docstring here to explain that it's checking for existing service and env tags and not overwriting them